examples/custom-indexer/rust/remote-reader.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Mysten Labs, Inc.
+// Copyright (c) Sui Foundation.
```

Comment:
```suggestion
// Copyright (c) Mysten Labs, Inc.
```
Don't think the CI allows it this way.

Reply:
Okay let me change that